docs/app/logs.md

```diff
@@ -1,7 +1,7 @@
 title: Sematext App - Logs
 description: Sematext App is a modern monitoring, log management, transaction tracing, and real user monitoring system that aggregates and structures your logs across various languages and platforms. It is a suite of products that combine high-quality logging experience with other monitoring and alerting devops tools helping fix production issues
 
-[Sematext logging platform](https://sematext.com/logsene/) provides both ability to create your own queries through its [Elasticsearch API](/docs/logs/search-through-the-elasticsearch-api/) as well as rich, yet simple query syntax very much like the [query syntax](/docs/logs/search-syntax/) used by Google. 
+[Sematext logging platform](https://sematext.com/logsene/) provides both ability to create your own queries through its [Sematext API](/docs/logs/search-through-the-sematext-api/) which is compatible with both Elasticsearch and Opensearch. It offers rich, yet simple query syntax very much like the [query syntax](/docs/logs/search-syntax/) used by Google. 
```

Comment:
Don't hate me @fulyauluturk but is Opensearch ==> OpenSearch everywhere (other than URLs) possible with a search/replace? That's the proper casing.